app/assets/javascripts/discourse/app/helpers/bound-avatar.js

```diff
@@ -1,14 +1,16 @@
+import { htmlSafe } from "@ember/template";
 import { addExtraUserClasses } from "discourse/helpers/user-avatar";
 import { avatarImg } from "discourse/lib/utilities";
-import { get } from "@ember/object";
-import { htmlHelper } from "discourse-common/lib/helpers";
 import { isEmpty } from "@ember/utils";
 
-export default htmlHelper((user, size) => {
+export default function boundAvatar(user, size) {
   if (isEmpty(user)) {
-    return "<div class='avatar-placeholder'></div>";
+    return htmlSafe("<div class='avatar-placeholder'></div>");
   }
 
-  const avatarTemplate = get(user, "avatar_template");
-  return avatarImg(addExtraUserClasses(user, { size, avatarTemplate }));
-});
+  return htmlSafe(
```

Comment:
I think we should move this `htmlSafe()` call into the `avatarImg` function. Better to keep the the `htmlSafe` declaration close to where the html is actually being generated.